internals/osutil/io_test.go

```diff
@@ -84,6 +84,9 @@ func (ts *AtomicWriteTestSuite) TestAtomicWriteFileSymlinkNoFollow(c *C) {
 	c.Assert(os.Chmod(rodir, 0500), IsNil)
 	defer os.Chmod(rodir, 0700)
 
+	if os.Getuid() == 0 {
+		c.Skip("requires running as non-root users")
```

Comment:
Grammar nit (here and elsewhere):
```suggestion
		c.Skip("requires running as non-root user")
```